Modules/Video/BridgeOpenCV/CMakeLists.txt

```diff
@@ -19,11 +19,15 @@ set(ITKVideoBridgeOpenCV_LIBRARIES ITKVideoBridgeOpenCV)
 
 # When this module is loaded by an app, load OpenCV too.
 set(ITKVideoBridgeOpenCV_EXPORT_CODE_INSTALL "
+set(OpenCV_CUDA ${OpenCV_CUDA})
```

Comment:
Strictly speaking, this is not required to fix the build issue but since it is documented along side the `OpenCV_STATIC` one, it is sensible to include it.

See https://github.com/opencv/opencv/blob/76aff8478883858f0e46746044348ebb16dc3c67/cmake/templates/OpenCVConfig.root-WIN32.cmake.in#L33-L35

Last, since both `OpenCV_CUDA` and `OpenCV_STATIC` are Windows specific option, we could conditionally include them in the config code using the same logic as in `OpenCVGenConfig.cmake` [^1].

Instead, let's just include a comment.

[^1]: https://github.com/opencv/opencv/blob/76aff8478883858f0e46746044348ebb16dc3c67/cmake/OpenCVGenConfig.cmake#L129-L137

Reply:
I've updated the commit message and added the comments about them being Windows specific options